example/src/enums.ts

```diff
@@ -0,0 +1,68 @@
+/** An integer enum. */
```

Comment:
I don't think we need this many enums, enums can mix and match between numeric/string enums, so those can be combined, and there's apparently no rendering difference between const and non-const enums, so we can probably drop that one too.